node/peer_states.js

```diff
@@ -22,13 +22,65 @@
 
 var inherits = require('util').inherits;
 
+var errors = require('./errors');
 var TChannelPeerState = require('./peer_state');
 
 module.exports.TChannelPeerHealthyState = TChannelPeerHealthyState;
+module.exports.TChannelPeerUnhealthyState = TChannelPeerUnhealthyState;
 
+/*
+ * Collectively, the health states receive additional options through the peer
+ * options:
+ * - maxErrorRate (error rate to go from healthy to unhealthy)
+ * - minResponseCount (response count to go from unhealthy to healthy)
+ * - TODO
+ *
+ * They also inherit:
+ * - channel.timers
+ * - channel.random
+ */
+
+var symptoms = {
+    'BadRequest': false, // not an indicator of bad health
+    'Cancelled': false, // not an indicator of bad health
```

Comment:
There are, but as it happens there is nothing to be gained by a level of indirection through that lookup table. We are getting the code as a string from the error classifier.